### ISSUE TITLE
Is matrix a required prop on View?

### ISSUE THREAD

**Issue by drock07** [react-paper/react-paper-bindings#12]
I can't figure out where View's matrix property fits into things. View's propTypes doesn't list it as required, but I get an error when I don't include it as a property on View. Specifically the error happens at line 82 of View.js. 
```
var x = matrix.x,
      y = matrix.y,
      zoom = matrix.zoom;
```

The error says "Cannot read property 'x' of undefined."

Is matrix supposed to be calculated from the width and height properties? If so, where does that happen? What am I missing?

**Comment by HriBB** [comment]
Hey @drock07 

Sorry for somewhat late reply, I am on a short holidays ATM.

You're right, matrix is confusing. View zoom and pan management should be part of the user land. Next version won't require it anymore, as it should be very easy to implement it as a HOC.

For now, you can just pass an object such as this, and it should work:
```
const matrix = {
  sx: 0,
  sy: 0,
  tx: 0,
  ty: 0,
  x: 0,
  y: 0,
  zoom: 1,
}
```
For more info, take a look at the [withMoveTool](https://github.com/HriBB/react-paper-bindings/blob/master/demo/src/Paper/hoc/withMoveTool.js#L11-L19) HOC from the demo.

And let me know if it works ;)

**Comment by HriBB** [comment]
@drock07 I released version 1.1.0, which does not require `matrix` anymore. Closing this issue, please reopen, if there is still a problem.
